db/db.go

```diff
@@ -0,0 +1,72 @@
+package db
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/datastore"
+)
+
+var (
+	candidateRolesCollection = "candidateRoles"
+	statesCollection         = "states"
+)
+
+type candidateType struct {
+	Role string
+}
+
+type state struct {
+	State  string
+	Cities []string
+}
+
+//Client manages all iteractions with mongodb
+type Client struct {
+	client *datastore.Client
+}
+
+// NewClient retuns a new Client
+func NewClient(client *datastore.Client) *Client {
+	return &Client{
+		client: client,
+	}
+}
+
+// GetCandidateRoles returns a list with candidate roles
+func (c *Client) GetCandidateRoles() ([]string, error) {
```

Comment:
Acho que esse método aqui só é necessário se ele for dinâmico com relação a disponibilidade. Explicando: se quando só existirem vereadores disponíveis ele só retornar "vereador".

 Se for para fazer uma consulta no BD e retornar 2 ou 3 registros, acho que o retorno deve ser hardcoded. Lembrando que isso conta como consultas no banco e seremos cobrados por isso.